### DISCUSSION TITLE
ripgrep support

### DISCUSSION THREAD

**Discussion by Alessandro-Barbieri** [PatrickF1/fzf.fish#187] [Ideas]
Is it possible to add ripgrep support as an fd alternative?

**Comment by PatrickF1** [comment]
Hi, that's an interesting idea. I had considered it but fd's API is more suited for listing files. I do have a feature on my todo list to implement a different function using ripgrep though. Does it cover your use case? https://github.com/PatrickF1/fzf.fish/pull/118
